spec/vsphere_vm_clone_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'chef/knife/vsphere_vm_clone'
+
+describe Chef::Knife::VsphereVmClone do
+  let(:datacenter) { double('Datacenter') }
+  let(:vim) { double('VimConnection', serviceContent: service_content) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.